gtk3/README.md

```diff
@@ -16,9 +16,9 @@ Ruby/GTK3 is a Ruby binding of GTK+ 3.
 
 ## Tutorials, examples:
 
-* gtk3/sample/misc: a lot of simple examples.
-* gtk3/sample/gtk-demo: the ruby version of `gtk-demo`.
-* gtk3/sample/tutorial: the ruby version of the [offical Gtk3 tutorial "getting started"](https://developer.gnome.org/gtk3/stable/gtk-getting-started.html).
+* [gtk3/sample/misc](/gtk3/sample/misc): a lot of simple examples.
+* [gtk3/sample/gtk-demo](/gtk3/sample/gtk-demo): the ruby version of `gtk-demo`.
```

Comment:
```suggestion
* [sample/gtk-demo](sample/gtk-demo/): the ruby version of `gtk-demo`.
```

Reply:
I can remove the gtk3/ prefix in the text, but I recommend keeping the absolute path in the link so that it can be copied/moved around the repo without breakage (from my repo documentation experience).